Source/CMakeLists.txt

```diff
@@ -1,3 +1,8 @@
+option(DOLPHIN_CXX_FLAGS "Flags used to compile Dolphin-only sources" "")
+if(DOLPHIN_CXX_FLAGS)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${DOLPHIN_CXX_FLAGS}")
```

Comment:
`add_compile_options()`?

Reply:
I thought about it, but then the interface becomes "pass these options separated by semicolons," rather than the much more widely-known "pass these options as you would on the command line."